example/chat/src/App.js

```diff
@@ -1,28 +1,33 @@
-import React, { Component } from 'react';
-import Room from './Room';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Room } from './Room';
 
-class App extends Component {
-    constructor(props) {
-        super(props);
+export const Input = styled.div`
```

Comment:
This probably doesnt need to be exported

```suggestion
const Input = styled.div`
```